hr_timesheet_sheet/tests/test_hr_timesheet_sheet.py

```diff
@@ -321,8 +321,12 @@ def test_5(self):
                 {'unit_amount': -1.0}, update=True))
         line.onchange_unit_amount()
         sheet._onchange_dates_or_timesheets()
-        self.assertEqual(len(sheet.line_ids), 0)
-        self.assertEqual(len(sheet.timesheet_ids), 0)
+        if line._negative_unit_amount():
+            self.assertEqual(len(sheet.line_ids), 7)
+            self.assertEqual(len(sheet.timesheet_ids), 1)
+        else:
+            self.assertEqual(len(sheet.line_ids), 0)
```

Comment:
Why do you need this `if _negative_unit_amount(): - else:` block in `hr_timesheet_sheet`?
This block and the similar others should be moved to the tests of the other module.

This way you can also avoid to define an empty method `_negative_unit_amount()` in module `hr_timesheet_sheet`.

Reply:
It's because I twas too lazy to copy the tests haha